chapters/classes.tex

```diff
@@ -1500,6 +1500,15 @@ \subsection{Real Type}\label{real-type}
 \index{unbounded@\robustinline{unbounded}!attribute of \robustinline{Real}}%
 \index{stateSelect@\robustinline{stateSelect}!attribute of \robustinline{Real}}
 
+The default \lstinline!min! and \lstinline!max!-attributes are the minimum and maximum representable finite floating point numbers of the underlying type.
+The value must be a finite floating point number, and thus the default default \lstinline!min! and \lstinline!max!-attributes do not further constrain the value.
+
+\begin{nonnormative}
+The minimum representable floating point number is usually the negation of the maximum representable floating point number, and should not be confused with the minimum positive one.
```

Comment:
Perhaps better to avoid _one_ in a complicated sentence like this?
```suggestion
Of the representable floating point numbers, the minimum number is usually the negation of the maximum number, and should not be confused with the minimum positive number.
```